src/components/scenes/TransactionListScene.js

```diff
@@ -121,8 +122,43 @@ class TransactionListComponent extends React.PureComponent<Props, State> {
 
   emptySection = () => [{ title: s.strings.transaction_list_search_no_result, data: [] }]
 
-  renderEmptyComponent = () =>
-    this.props.numTransactions ? <EmptyLoader /> : <BuyCrypto walletId={this.props.selectedWalletId} currencyCode={this.props.selectedCurrencyCode} />
```

Comment:
TODO: This file did not get merged completely right after the last merge conflict resolution. This file should be using the BuyCrypto component to stay in sync with the ui2 branch.